app/views/payments/index.html.erb

```diff
@@ -0,0 +1,37 @@
+<div class="container vh-100 d-flex flex-column">
+<div class="d-flex justify-content-around fixed-top pt-2 align-items-center heading-actions">
+<%= link_to groups_path do %>
+  <span class="arrow"><i class="uil uil-arrow-left icon"></i></span>
+<% end %>
+<h1>TRANSACTIONS</h1>
+<i class="uil uil-search"></i>
+</div>
+
+<div>
+<h2>Payments for <%= @group.name %></h2>
+<p class="price">Total Amount: $<%= number_with_precision(@total_amount, precision: 2) %></p>
+
+</div>
+
+<table class="table">
+<thead>
+    <tr>
+      <th scope="col">Transaction</th>
+      <th scope="col">Amount</th>
+    </tr>
+  </thead>
+  <tbody>
+  <% @payments.each do |payment| %>
+    <tr>   
+      <td><%= payment.name %></td>
+      <td>$<%= number_with_precision(payment.amount, precision: 2) %></td>
+      
+      
+    </tr>
+    <% end %>
+  </tbody>
+</table>
+
+<%= link_to 'Add a New Transaction', new_group_payment_path(@group), class: "btn btn-primary mt-2" %>
+
+</div>
```

Comment:
- [ ] [REQUIRED] You have done a great job so far @ProgramKingAbel , however, it is a requirement to abide with the design template  provided in terms of colour,and the general layout. Let's make sure that the button to add transactions is green and fixed at the bottom of the screen just as it it in the design template provided.

Your Implementation | Template Design
:--------------------: | :-------------------:
<img width="302" alt="Screenshot 2023-09-29 at 22 06 16" src="https://github.com/ProgramKingAbel/E-nstaPay/assets/58692590/279530d5-dee9-48ab-8793-fcbd3bfd4c75"> | <img width="405" alt="Screenshot 2023-09-29 at 22 05 34" src="https://github.com/ProgramKingAbel/E-nstaPay/assets/58692590/4ebbf360-a5cd-4aee-b665-68939fc917c8">